src/Functions/FunctionHelpers.cpp

```diff
@@ -128,12 +127,12 @@ void validateArgumentsImpl(const IFunction & func,
         const auto & descriptor = descriptors[i];
         if (int error_code = descriptor.isValid(arg.type, arg.column); error_code != 0)
             throw Exception(error_code,
-                            "Illegal type of argument #{}{} of function {}{}{}",
-                            argument_offset + i + 1,    // +1 is for human-friendly 1-based indexing
-                            (descriptor.argument_name ? " '" + std::string(descriptor.argument_name) + "'" : String{}),
+                            "Illegal type of argument {}{} of function '{}' {}{}",
```

Comment:
Sorry missed the first time, but now we get
`Illegal type of argument 5th 'some_name' of function 'some function'`
did you want to move the number before the word `argument`?

Reply:
Will fix, tnx.